### ISSUE TITLE
Implement Anonymous Posting Option switch

### ISSUE THREAD

**Issue by bmirzo** [CMU-313/fall23-nodebb-qwerty-coders#8]
User story: Anonymous post function 

Create a feature that allows users to toggle between posting anonymously or with their identity.
Add a button or checkbox in the post creation interface to enable this option.

Estimated effort:
~ 4 hrs

Dependencies: No dependency for the UI, but the connection to the Model will require the actual functionality to be working properly, menaing that issue #10 should be finished before we make the connection.

Acceptance criteria: Manual test on local server

For sprint 1, the function would be implemented front-end based. Further features like different names and database might be postponed to sprint 2 or stretched
